packages/honeycomb-opentelemetry-web/src/honeycomb-otel-sdk.ts

```diff
@@ -28,6 +28,7 @@ export class HoneycombWebSDK extends WebSDK {
         ),
       );
     }
+    ``;
```

Comment:
is this for something?

Reply:
A typo! :D

Reply:
9147e28bd13fdc8d9f650d44aa64f06ee4678db3